tests/ludwig/utils/test_data_utils.py

```diff
@@ -169,3 +171,13 @@ def test_dataset_synthesizer_output_feature_decoder():
     }
     build_synthetic_dataset_df(dataset_size=100, config=config)
     LudwigModel(config=config, logging_level=logging.INFO)
+
+
+def test_chunking():
```

Comment:
@abidwael is it this simple for a public dataset?

Reply:
This should work. Could you parametrize this test? You can pass in the paths as arguments, then use `ludwig.utils.data_utils.figure_data_format_dataset` to get the file type and use the corresponding read function based on that.

Reply:
Assuming it works, will come back and improve this test, right now want to get it in ASAP.